implementations/rust/ockam/ockam_core/src/vault/types.rs

```diff
@@ -244,6 +244,15 @@ impl KeyAttributes {
             length,
         }
     }
+
+    /// Create default key attributes
+    pub fn default_attributes() -> Self {
```

Comment:
I think being explicit when you generate a secret could be very useful

Reply:
Do you mean that we should actually *not* have such a function?

Reply:
Yes